.github/workflows/lint_random_files.yml

```diff
@@ -335,31 +335,31 @@ jobs:
           done
 
           if [[ -n "${files}" ]]; then
-            make lint-javascript-files FIX="${FIX}" FAST_FAIL=0 FILES="${files}" 2>&1 | tee -a "$ERR_FILE"
+            make lint-javascript-files FIX="${FIX}" FAST_FAIL=0 FILES="${files}" ESLINT_ERROR_LOG="$ERR_FILE"
```

Comment:
To avoid redundancy, I would suggest creating a small helper function which performs the retry logic and then redirects to `ERR_FILE` on lint failure.